checkov/common/output/csv.py

```diff
@@ -157,8 +159,23 @@ def persist_report_oss_packages(self, file_name: str, is_api_key: bool, output_p
             is_api_key=is_api_key,
         )
 
+    @staticmethod
+    def arrange_rows(rows: list[dict[str, Any]]) -> None:
+        # we search for formatted rows and covert them back into csv formatted file.
+        for row in rows:
+            for key in row.keys():
+                val = str(row[key])
+
+                if is_raw_formatted(val):
+                    row[key] = ', '.join(format_string_to_licenses(val))
+                    val = str(row[key])
+                row[key] = val[1:-1] if val.startswith('"') and val.endswith('"') else row[key]
+                row[key] = '' if val == 'None' else row[key]
```

Comment:
```suggestion
            for key, value in row.items():
                val = str(value)

                if is_raw_formatted(val):
                    val = ', '.join(format_string_to_licenses(val))
                val = val[1:-1] if val.startswith('"') and val.endswith('"') else val
                row[key] = '' if val == 'None' else val
```
no need to always write to `row[key]` access it